docs/troubleshooting.rst

```diff
@@ -96,9 +96,13 @@ As an example, here is a working config :
             python-version: ${{ matrix.python }}
         - name: setup ${{ matrix.os }}
           run: |
-            sudo apt install libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0
+            sudo apt install libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 x11-utils
             /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1920x1200x24 -ac +extension GLX
 
+``tlambert03/setup-qt-libs``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Instead manually curate list of used packages you may use ``tlambert03/setup-qt-libs`` github action: https://github.com/tlambert03/setup-qt-libs
```

Comment:
Took a look and the action seems solid: https://github.com/tlambert03/setup-qt-libs/blob/main/index.js

Reply:
it is created for [napari](https://github.com/napari/napari) project to simplify life with [multiple workflows](https://github.com/napari/napari/tree/main/.github/workflows) that requires Qt and also as a way to reduce overhead on plugin creators when they use [cookiecutter](https://github.com/napari/cookiecutter-napari-plugin/blob/b5ecc6c2af3082bc90ae1c8a7ec7c861fd19b052/%7B%7Bcookiecutter.plugin_name%7D%7D/.github/workflows/test_and_deploy.yml#L37)

So as long `napari` projects live it should be maintained as tlambert03 is one of the main persons in this project.